embrace-android-delivery/src/main/kotlin/io/embrace/android/embracesdk/internal/delivery/execution/HttpMethodV2.kt

```diff
@@ -0,0 +1,21 @@
+package io.embrace.android.embracesdk.internal.delivery.execution
+
+import java.util.Locale
+
+enum class HttpMethodV2 {
```

Comment:
Why do we need this? This service is only going to POST at this point, and eventually GET. I don't think we need an abstraction over the method? And if we do, we may want to just have separate POST and GET request objects that set the method explicitly and call it a day

Reply:
we don't! at first I thought the server expected the field, but it's just used to modify the connection